devs/samples/display-dimmer.ts

```diff
@@ -20,7 +20,7 @@ ds.Led.prototype.setAllColors = async function (r, g, b) {
     await this.setAll(r, g, b)
 }
 
-ledD.onConnected(async () => {
+ledD.binding().subscribe(async () => {
     await ledD.setAllColors(0.9, 1, 0)
```

Comment:
should this check be `async (bound) => { if (bound) await ledD... }` ? as it stands this will also run on disconnect